configs/spmd/dbscan.json

```diff
@@ -0,0 +1,44 @@
+{
+    "PARAMETERS_SETS": {
+        "implementations": {
+            "algorithm": [
+                { "library": "sklearnex", "device": "distributed", "num_procs": 2, "procs_per_node": 2 }
```

Comment:
In my opinion default value for num_procs and procs_per_nodes 6 will be a better choice sice pvc has 6 GPUs.